extensions/arc/src/ui/dashboards/postgres/postgresDashboard.ts

```diff
@@ -0,0 +1,51 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as loc from '../../../localizedConstants';
+import { ControllerModel } from '../../../models/controllerModel';
+import { PostgresModel } from '../../../models/postgresModel';
+import { PostgresOverviewPage } from './postgresOverviewPage';
+import { PostgresComputeStoragePage } from './postgresComputeStoragePage';
+import { PostgresConnectionStringsPage } from './postgresConnectionStringsPage';
+import { PostgresBackupPage } from './postgresBackupPage';
+import { PostgresPropertiesPage } from './postgresPropertiesPage';
+import { PostgresNetworkingPage } from './postgresNetworkingPage';
+import { Dashboard } from '../../components/dashboard';
+
+export class PostgresDashboard extends Dashboard {
+	constructor(title: string, private _controllerModel: ControllerModel, private _databaseModel: PostgresModel) {
+		super(title);
+	}
+
+	protected async registerTabs(modelView: azdata.ModelView): Promise<(azdata.DashboardTab | azdata.DashboardTabGroup)[]> {
+		await Promise.all([this._controllerModel.refresh(), this._databaseModel.refresh()]);
```

Comment:
We'll want to change this before we ship so that there's some indication that it's in a loading state - otherwise it might just be sitting there on an empty blank page for a while if the controller is slow to respond.